src/index.html

```diff
@@ -3,107 +3,181 @@
+          </section>
+      </div>
+    </section>
+
+    <div class="content_wrapper">
+        <section class="content_main">
+
+          <section class="section__intro">      
+            <div class="content-l_col content-l_col-1-2">
+              <h2>Know the process</h2>
+              <p>Buying a home can be a daunting process. If you're feeling overwhelmed, you're not alone. We've got tools and resources to help you every step of the way.</p>
+            </div>
+
+            <div class="content-l_col content-l_col-1-2">
+              <img src="http://placekitten.com/380/200" alt="Illustration of the home-buying process">
+          </div>
+          </section>
+
+          <section class="section__phase grid_column__top-divider block__flush-top">
+            <figure class="phase_illu content-l_col content-l_col-1-4">
+              <img src="http://placekitten.com/g/150/150" alt="Illustration of the home-buying process">
+            </figure>
+
+            <div class="phase_intro content-l_col content-l_col-1-2">
+              <div class="content-l_col content-l_col-1 content-l__large-gutters">
+              <h3>Prepare to shop</h3>
+              <p>Not sure how to get started, how much you can afford, or what to expect when buying a home? Set yourself up for success with a little bit of preparation.</p>
+              <p><a class="go-link" href="{{ base_url }}process/prepare">Learn more</a></p>
+              </div>
+            </div>
+
+             <div class="phase_tools content-l_col content-l_col-1-4 block__sub grid_column__left-divider">
+                <h6 class="block__sub block__flush-top">Key tools</h6>
+                <ul class="list__unstyled block__flush-top">
+                  <li class="block__sub block__flush-top"><a href="{{ base_url }}prepare-worksheets">Worksheet: clarify your goals</a></li>
+                  <li class="block__sub block__flush-top"><a href="{{ base_url }}check-rates">Check interest rates for your situation</a></li>
+                </ul>
+              </div>
+          </section>
+
+          <section class="section__phase grid_column__top-divider block__flush-top">
+            <figure class="phase_illu content-l_col content-l_col-1-4">
+              <img src="http://placekitten.com/g/150/150" alt="Illustration of the home-buying process">
+            </figure>
+
+            <div class="phase_intro content-l_col content-l_col-1-2">
+              <div class="content-l_col content-l_col-1 content-l__large-gutters">
+              <h3>Explore loan options</h3>
+              <p>You've got a pretty good idea of your priorities and budget, and you're ready to start home shopping in earnest. Now is the time to start exploring loan options and meeting with lenders.</p>
+              <p><a class="go-link" href="{{ base_url }}process/explore">Learn more</a></p>
+              </div>
```

Comment:
We can make the `go-link` a `jump-link` so it's more finger-friendly on mobile screens.